docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -164,6 +164,10 @@ When redirect handling is on, any 3xx response from upstream, that matches
 <envoy_v3_api_field_config.route.v3.InternalRedirectPolicy.redirect_response_codes>`
 is subject to the redirect being handled by Envoy.
 
+If Envoy is configured to internally redirect HTTP 303, it will dispatch the redirect with a
+bodiless HTTP GET regardless of the original HTTP method. For more information, see `RFC 7231
+Section 6.4.4 <https://tools.ietf.org/html/rfc7231#section-6.4.4>`_.
```

Comment:
Shouldn't HEAD stay HEAD, or am I misreading that section?

Reply:
I think this is the relevant statement?
```﻿﻿
﻿                                               ﻿﻿A u﻿ser agent can perform
   a retrieval request targeting that URI (a GET or HEAD request if
   using HTTP), which might also be redirected, and present the eventual
   result as an answer to the original request.
```

I initially didn't pick up "HEAD should stay HEAD" from that, but that makes perfect sense to me since HEAD->GET changes the semantics of the initial request, almost an "upgrade" of sorts. I'll push an fix.